Cargo.lock

```diff
@@ -2222,7 +2222,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
  "num-integer 0.1.39 (registry+https://github.com/rust-lang/crates.io-index)",
  "num-traits 0.2.6 (registry+https://github.com/rust-lang/crates.io-index)",
- "rand 0.4.3 (registry+https://github.com/rust-lang/crates.io-index)",
+ "rand 0.3.22 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
This is probably downgrade from https://github.com/rust-lang/cargo/issues/5702, do we want to revert it?

Reply:
I'm surprised we're not using `cargo build --frozen` in our CI to detect that kind of problems.

Reply:
I guess `cargo update -p rand:0.3.22` should fix that. `cargo check --frozen` won't fail, the downgrade only happens if you do `cargo update -p <some dep>`.